### PR TITLE
Add AWX project to config

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Uses the new text-based reaction suggesting that Hebbot has.

@gundalow are those usual_reporters right?
@cybette how do you feel about a tools/projects section? We may get a bit more from Edge, EDA, etc

### CODE REVIEW

File: bots/newsbot-config.toml [hunk 1]
```diff
@@ -82,6 +82,18 @@ title = 'THE ANSIBLE TEAM IS HIRING'
...
 usual_reporters = []
 
+[[projects]]
+emoji = 'AWX'
+name = 'AWX'
+title = 'AWX Project'
+description = 'AWX provides a web-based user interface, REST API, and task engine built on top of Ansible.'
+website = 'https://github.com/ansible/awx'
+default_section = 'general'
+usual_reporters =
```
Comment (cybette) [inline bots/newsbot-config.toml:92]: does it need an opening `[`

**Comment by cybette** [comment]
@GregSutcliffe I think it makes sense to have a "Project Updates" section when they have something other than release updates, as those don't fall under "Collection Updates" but not quite "Community Updates" either.

File: bots/newsbot-config.toml (hunk 1 above)
Comment (GregSutcliffe) [reply bots/newsbot-config.toml:92]: whoops, silly me. ty

**Comment by GregSutcliffe** [comment]
@cybette added a section, see what you think

File: bots/newsbot-config.toml [hunk 2]
```diff
@@ -40,48 +40,67 @@ title = 'MAJOR NEW RELEASES'
 order = 102
 usual_reporters = []
 
+[[sections]]
+emoji = '🛠️' # :hammer_and_wrench:
+name = 'projects'
+title = 'Project Updates'
```
Comment (cybette) [inline bots/newsbot-config.toml:46]: Upper case for consistency (looks better in the current Bullhorn formatting). The rest LGTM!

Comment (GregSutcliffe) [reply bots/newsbot-config.toml:46]: :+1:

**Comment by cybette** [comment]
Looks great, thanks @GregSutcliffe !

**Pull request merged**